Modules/Invoices/Controllers/InvoicesAjaxController.php

```
@@ -25,25 +25,13 @@
-    public function __construct(
-        InvoiceService $invoiceService,
-        InvoiceItemService $invoiceItemService,
-        UserService $userService,
-        ClientService $clientService,
-        InvoicesRecurringService $invoicesRecurringService
+{    public function __construct(
+        protected InvoiceService $invoiceService,
+        protected InvoiceItemService $invoiceItemService,
+        protected UserService $userService,
+        protected ClientService $clientService,
+        protected InvoicesRecurringService $invoicesRecurringService
```

PHPMD (2.15.0)

Avoid excessively long variable names like $invoicesRecurringService. Keep variable name length under 20. (undefined)

(LongVariable)